pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -288,8 +287,9 @@ func (p *Provider) hydrateCache(ctx context.Context) {
 		return true
 	}); err != nil {
 		logging.FromContext(ctx).Errorf(fmt.Sprintf("Unable to hydrate the AWS launch template cache, %s", err))
+	} else {
+		logging.FromContext(ctx).With("item-count", p.cache.ItemCount()).Debugf("hydrated launch template cache")
```

Comment:
```suggestion
		logging.FromContext(ctx).With("count", p.cache.ItemCount()).Debugf("hydrated launch template cache")
```

Maybe just count since it's clear what the count is referring to?